packages/app/src/navigation/SidebarNavigation.cy.tsx

```diff
@@ -203,6 +213,8 @@ describe('SidebarNavigation', () => {
 
       mountComponent({ online: false })
 
+      cy.tick(1000) //wait for debounce
```

Comment:
Weird this is required, was thinking that Cypress should auto wait - it's one of the main features. Does it fail without this?

Reply:
The `cy.tick` is needed here because of the `cy.clock` that is called earlier to make sure the clock is within the `IATR_RELEASE` window.  The badge has a check that will no longer show the "new" part two months after the IATR release date.